scim2-sdk-common/src/main/java/com/unboundid/scim2/common/messages/PatchOperation.java

```diff
@@ -49,7 +49,48 @@
 import static com.unboundid.scim2.common.utils.StaticUtils.toList;
 
 /**
- * An individual patch operation.
+ * This class represents a SCIM 2 PATCH operation. A patch operation is a
+ * component of a {@link PatchRequest}, and it represents an individual update
+ * to a SCIM resource. A patch operation must be one of the following types:
+ * <ul>
+ *   <li> add
+ *   <li> remove
+ *   <li> replace
+ * </ul>
+ *
+ * An {@code add} operation will add new attribute data. A {@code remove}
+ * operation will delete the existing value(s) on an attribute. A
+ * {@code replace} operation will overwrite any existing values of an attribute.
+ * <br><br>
+ *
```

Comment:
Do you wish/need to mention replace with an empty array since you recently made that change?